packages/taiko-client/driver/chain_syncer/event/blocks_inserter/pacaya.go

```diff
@@ -98,15 +99,11 @@ func (i *BlocksInserterPacaya) InsertBlocks(
 			len(meta.GetBlobHashes()) != 0,
 			true,
 		)
-		parent          *types.Header
-		lastPayloadData *engine.ExecutableData
+		lastestSeenProposal = &encoding.LastSeenProposal{TaikoProposalMetaData: metadata}
```

Comment:
typo "lastestSeen" should be "latestSeen"

Reply:
fixed